README.md

```diff
@@ -2,40 +2,148 @@
+        - Press `R` to use last selected Poké Ball
+        - Hold `R` to change selection with the D-Pad.
+    - Faster battle intro
+        - Message and animation/cry happens at the same time
+    - Faster HP drain.
+    - Battle Debug menu.
+        - Accessed by pressing `Select` on the "Fight/Bag/Pokémon/Run" menu.
+    - Option to use AI flags in wild Pokémon battles
+    - FRLG/Gen4+ whiteout money calculation.
+    - Configurable experience settings
+        - Experience on catch.
+        - Splitting experience.
+        - Trainer experience.
+        - Scaled experience.
+        - Unevolved experience boost.
+    - Frostbite.
+        - Doesn't replace freezing unless a config is enabled, so you can mix and match.
+    - Critical capture.
+    - Removed badge boosts (configurable).
+    - Recalculating stats at the end of every battle.
+    - Level 100 Pokémon can earn EVs.
+    - Inverse battle support.
+    - TONS of other features listed [here](/include/config/battle.h).
+- Full Trainer customization
+    - Nickname, EVs, IVs, moves, ability, ball, friendship, nature, gender, shininess.
+    - Custom tag battle support (teaming up an NPC in a double battle).
+    - Sliding trainer messages.
+    - Upgraded Trainer AI
+        - Considers newer move effects.
+        - New flag options to let you customize the intelligence of your trainers.
+        - Faster calculations.
+    - Specify Poké Balls by Trainer class.
+- Pokémon Species from Generations 1-8.
+    - Option to disable unwanted generations.
+    - Updated sprites to DS style.
+    - Updated stats, types, abilities and egg groups (configurable).
+    - Updated Hoenn's Regional Dex to match ORAS'.
+    - Updated National Dex incorporating the new species.
+    - Sprite and animation visualizer.
+        - Accesible by pressing `Select` on a Pokémon's Summary screen.
+    - Gen4+ evolution methods, with some changes:
+        - Mossy Rock, Icy Rock and Magnetic Field locations match ORAS'.
+            - Leaf, Ice and Thunder Stones may also be used.
+        - Inkay just needs level 30 to evolve.
+            - You can't physically have both the RTC and gyroscope, so we skip this requirement.
+        - Sylveon uses Gen8+'s evolution method (friendship + Fairy Move).
+        - Option to use hold evolution items directly like stones.
+    - Hidden Abilities.
+        - Available via Ability Patch.
+        - Compatible with Ghoul's DexNav branch.
+    - All gender differences.
+        - Different icons for female Hippopotas and Hippowdon
+    - 3 Perfect IVs on Legendaries, Mythicals and Ultra Beasts
+- Breeding
+    - Incense Baby Pokémon now happen automatically (configurable).
+    - Level 1 eggs.
+    - Poké Ball inheriting.
+    - Egg Move Transfer, including Mirror Herb.
+    - Nature inheriting 100% of the time with Everstone
+    - Gen6+ Ability inheriting.
+- Items from newer Generations
+    - ***Gen 6+ Exp. Share*** (configurable)
+    - 
```

Comment:
Was something supposed to be here?

Reply:
Oh right, I wanted to add what items were added, or at least items added that are missing their functionality for now.

Reply:
Fixed.